### ISSUE TITLE
awxkit/awxkit/cli/docs - make documentation is broken (sphinx)

### ISSUE THREAD

**Issue by mrwiora** [ansible/awx#4879]
##### ISSUE TYPE
 - Bug Report

##### ENVIRONMENT
* AWX version: 7.0.0 (d2a5af44)
* AWX install method: docker
* Ansible version:  X.Y.Z
* Operating System: debian 10 on hostsystem (latest updates applied)
* Web Browser:

##### STEPS TO REPRODUCE

```
cd awxkit/awxkit/cli/docs
TOWER_HOST=https://awx.example.org TOWER_USERNAME=example TOWER_PASSWORD=secret make clean html
```

##### EXPECTED RESULTS

a working documentation

##### ACTUAL RESULTS

a lot of sphings build issues:
```
sphinx-build hosts: too few arguments
usage: sphinx-build activity_stream [-h] action ...

positional arguments:
  action
    list
    get
    modify
    delete

optional arguments:
  -h, --help  show this help message and exit
```
and a note, that contents.rst is missing
```
making output directory...
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 6 source files that are out of date
updating environment: 6 added, 0 changed, 0 removed
reading sources... [100%] usage

Sphinx error:
master file /root/awx/awxkit/awxkit/cli/docs/source/contents.rst not found
make: *** [Makefile:20: html] Error 2
```

adding this content.rst with a random content you will get:
```
making output directory...
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 7 source files that are out of date
updating environment: 7 added, 0 changed, 0 removed
reading sources... [100%] usage
looking for now-outdated files... none found
pickling environment... done
checking consistency... /root/awx/awxkit/awxkit/cli/docs/source/index.rst: WARNING: document isn't included in any toctree
done
preparing documents... done
writing output... [ 85%] reference
Exception occurred:
  File "/usr/local/lib/python2.7/dist-packages/docutils/writers/html4css1/__init__.py", line 642, in should_be_compact_paragraph
    first = isinstance(node.parent[0], nodes.label) # skip label
TypeError: 'NoneType' object has no attribute '__getitem__'
The full traceback has been saved in /tmp/sphinx-err-rychgG.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
make: *** [Makefile:20: html] Error 2

```

##### ADDITIONAL INFORMATION

```
sphinx-build --version
sphinx-build 1.8.5
```

**Comment by ryanpetrello** [comment]
You might want to get a newer `sphinx-build`?

```shell
sphinx-build --version
sphinx-build 2.2.0
```

I'm unable to reproduce this from a clean `git clone`:

```shell
~ git clone git@github.com:ansible/awx.git && cd awx
~ pip install -e ./awxkit
~ cd awxkit/awxkit/cli/docs
~ pip install sphinx sphinxcontrib-autoprogram
~ TOWER_HOST=https://awx.example.org TOWER_USERNAME=example TOWER_PASSWORD=secret make clean html
```

<img width="545" alt="image" src="https://user-images.githubusercontent.com/214912/66047310-8b7f3180-e4f5-11e9-942d-eafa2ca0d58c.png">

**Comment by ryanpetrello** [comment]
Also, are you running this under python3?  That might be it (this is only intended to work in python3, not python2).

**Comment by mrwiora** [comment]
@ryanpetrello thx!

@ public - for those having the same issue...:
actually I'm running debian 10 and pip is separated from pip3 - executing all commands from readme with pip3 and python3 brings the expected results:
```
pip3 install sphinx sphinxcontrib-autoprogram
pip3 install "https://github.com/ansible/awx/archive/$VERSION.tar.gz#egg=awxkit&subdirectory=awxkit"
TOWER_HOST=https://awx.example.org TOWER_USERNAME=example TOWER_PASSWORD=secret make clean html
cd build/html/ && python3 -m http.server
```
a crosscheck for the sphinx version shows:
```
sphinx-build --version
sphinx-build 2.2.0
```
so sphinx development for python2 has been stopped on version 1.8.5 while it is available on verison 2.2.0 on python3...
